ra_aid/llm.py

```
@@ -217,7 +242,7 @@ def initialize_llm(
 
 
 def initialize_expert_llm(
-    provider: str = "openai", model_name: str = "o1"
+    provider: str, model_name: str
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)